src/main/java/com/cburch/logisim/fpga/hdlgenerator/HDL.java

```diff
@@ -63,6 +81,41 @@ public static String assignOperator() {
     return isVHDL() ? " <= " : " = ";
   }
 
+  public static String equalOperator() {
+    return isVHDL() ? " = " : "==";
```

Comment:
Why do we have surrounding spaces only in one case? (multiple occurrences below)